src/bulk-load.ts

```diff
@@ -92,6 +92,13 @@ class RowTransform extends Transform {
 
     for (let i = 0; i < this.columns.length; i++) {
       const c = this.columns[i];
+      if (this.bulkLoad.options.validateBulkLoadParameters) {
+        const error = c.type.validate(row[i]);
+
+        if (error instanceof TypeError) {
+          return callback(error);
```

Comment:
Hi @arthurschreiber, since the error is not thrown here, we need additional handling for the error under "addRow" method, right?